porcupine/plugins/filemanager.py

```diff
@@ -78,7 +109,7 @@ def validate_name(*junk: object) -> None:
             ok_button.config(state="disabled")
             return
 
-        if possible_new_path.exists():
+        if choice.get() == 1 and possible_new_path.exists():
```

Comment:
It's not clear what the integer `1` does here. Instead of values `0` and `1`, it might be simpler to use values `"overwrite"` and `"change_name"`, or something like that. You will need to change `choice` to be a `StringVar`.

Reply:
Actually, since it has two values, a `BooleanVar` would make sense the most. It could be named so that it is obvious which choice is which, e.g. `overwrite_var`.